operator/src/main/resources/scripts/wdt_mii_filter.py

```diff
@@ -0,0 +1,546 @@
+# Copyright (c) 2018, 2021, Oracle and/or its affiliates.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+#
+# ------------
+# Description:
+# ------------
+#  This is a WDT filter for handling WLS configuration overrides.
+#
+
+import inspect
+import os
+import sys
```

Comment:
General comment #1: I suspect the life-cycle and/or architecture documentation describes the propagation of internal sit-cfg.  If so, then these docs should be updated.

General comment #2: Does this change lift any of the restrictions detailed in 'runtime-updates.md' for MII?  If so, then the doc should be updated. Note that if the update includes adding support for shrinking a cluster we (a) may want to have integration tests before officially documenting support, (b) may want to make sure the documentation clearly states that JTA and JMS are not particularly amenable to such shrinks.